css/index.css

```diff
@@ -71,6 +71,18 @@ p {
     margin: 0 auto;
 }
 
+header {
+    display: flex;
+    align-items: baseline;
+    margin: 20px 0;
+}
+
+header nav a {
+    text-decoration: none;
+    color: black;
+    margin: 35px;
```

Comment:
You can space out the a tags with flexbox instead of margin if you add `display: flex;` to the nav tag, but margins work fine for today!